ST_params.c

```diff
@@ -855,10 +866,15 @@ static void _rgroup_init( void) {
    char name[80]; /* plenty of space to read possibly long name*/
 
    /* input variables*/
-   Int estab, stretch, xres, turnon, extirp, mort, estann,
-       styr, killyr, killfreq_startyr,veg_prod_type, grazingfreq_startyr;
-   RealF space, density, slow, killfreq, xgro,
-         nslope, nint, wslope, wint, dslope, dint,prop_killed, prop_recovered,grazing_frq, prop_grazing ;
+   Int estab, stretch, xres, turnon, estann,
+       styr, veg_prod_type;
+   RealF space, density, slow, 
+         nslope, nint, wslope, wint, dslope, dint, xgro;
+   /* input variables related to disturbances */
+   Int extirp, mort,
```

Comment:
`mort` actually has nothing to do with disturbance. it implements age-independent mortality, which is a different form of mortality than mortality from disturbance....

Reply:
I'll move mort back in with the variables right above this comment.

Reply:
Commit 983faa7 resolves this.